### PR TITLE
Add USWDS identifier, edit footer

### LINKED ISSUE #2474: Add USWDS Identifier component
Per [required web content and links – Digital.gov](https://digital.gov/resources/required-web-content-and-links/), we need the USWDS Identifier on our page (details in the link).

## Notes

- https://designsystem.digital.gov/components/identifier/
- [M-24-08 Strengthening Digital Accessibility and the Management of Section 508 of the Rehabilitation Act](https://www.whitehouse.gov/omb/management/ofcio/m-24-08-strengthening-digital-accessibility-and-the-management-of-section-508-of-the-rehabilitation-act/)

## Acceptance Criteria

- [ ] Identifier is added
- [ ] [GSA EDX check](https://chromewebstore.google.com/detail/gsa-edx/inphcfjkfegopafbnfkclnkikkdehjfa) passes with no issues

### PR DESCRIPTION
**Pull request by drewbo**
## Changes proposed in this pull request:
- Close #2474 
- Edit footer information to avoid duplication

## Notes
The footer is a bit outdated in terms of both content and design and could use a quick refresh. There are good examples from other TTS sites:
- https://www.login.gov/
- https://search.gov/
- https://digitalcorps.gsa.gov/
- https://data.gov/

I'd like to keep this PR fairly tightly scoped to getting in necessary information and marginally improved design with the notion that we can improve this more in the future (but need to get the required Identifier component in relatively quickly)

## Security Considerations
None

### CODE REVIEW

**Review by hursey013 — DISMISSED** [review]
I'll no longer be able to use cloud.gov of as an example of a site that's missing a bunch of required links! Well done! ✨

**Comment by hursey013** [comment]
Screenshot 2024-03-12 at 3 01 59 PM

File: _assets/images/content/cloud-gov-logo.svg
Comment (sknep) [inline _assets/images/content/cloud-gov-logo.svg:1]: The star was transparent; it needs to be white

File: _includes/identifier.html [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+      aria-label="U.S. government information and services,"
+    >
+      <div class="usa-identifier__container">
+        <div class="usa-identifier__usagov-description">
+          Looking for U.S. government information and services?
+        </div>
+        <a href="https://www.usa.gov/" class="usa-link"> Visit USA.gov </a>
+      </div>
+    </section>
+  </div>
```
Comment (sknep) [inline _includes/identifier.html:111]: Screenshot 2024-03-13 at 11 57 32 AM

There's a small border on the top to separate it from the page background of the docs section. Feel free to add more links as you like

**Comment by sknep** [comment]
I think this is now ready to be approved!

**Review by hursey013 — APPROVED** [review]
Can confirm the identifier is a required element for GSA websites, and the inclusion of it ensures we have all of the required links now 🎉

**Pull request merged**